composer.json

```diff
@@ -49,6 +49,7 @@
         "php": ">=5.3.3"
     },
     "require-dev": {
+        "hoa/math": "0.15.05.29",
```

Comment:
Oh yeh. I thought we were going to make it an optional dep to use all phpseclib features, but, yeh, it really is only used in the tests. ;)